.github/workflows/pull_request.yml

```diff
@@ -10,9 +10,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-python@v1
+      - uses: actions/setup-python@v2
         with:
-          python-version: 3.8
+          python-version: 3.9
```

Comment:
We should update this action to use a variable here (e.g. ${{ env.PYTHON_VERSION }}) rather than hardcoding to make it consistent with the other repos.

Reply:
Done, thanks!